Makefile

```diff
@@ -1,8 +1,8 @@
 unit:
-	mvn test -Dcucumber.filter.tags="@unit.offline or @unit.algod or @unit.indexer or @unit.rekey or @unit.indexer.rekey or @unit.transactions or @unit.transactions.keyreg or @unit.responses or @unit.applications or @unit.applications.boxes or @unit.dryrun or @unit.tealsign or @unit.responses.messagepack or @unit.responses.231 or @unit.responses.messagepack.231 or @unit.feetest or @unit.indexer.logs or @unit.abijson or @unit.abijson.byname or @unit.atomic_transaction_composer or @unit.transactions.payment or @unit.responses.unlimited_assets or @unit.algod.ledger_refactoring or @unit.indexer.ledger_refactoring or @unit.dryrun.trace.application"
+	mvn test -Dcucumber.filter.tags="@unit.offline or @unit.algod or @unit.indexer or @unit.rekey or @unit.indexer.rekey or @unit.transactions or @unit.transactions.keyreg or @unit.responses or @unit.applications or @unit.applications.boxes or @unit.dryrun or @unit.tealsign or @unit.responses.messagepack or @unit.responses.231 or @unit.responses.messagepack.231 or @unit.feetest or @unit.indexer.logs or @unit.abijson or @unit.abijson.byname or @unit.atomic_transaction_composer or @unit.transactions.payment or @unit.responses.unlimited_assets or @unit.algod.ledger_refactoring or @unit.indexer.ledger_refactoring or @unit.dryrun.trace.application or @unit.sourcemap"
 
 integration:
-	mvn test -Dcucumber.filter.tags="@algod or @assets or @auction or @kmd or @send or @send.keyregtxn or @indexer or @rekey or @applications.verified or @applications or @applications.boxes or @compile or @dryrun or @indexer.applications or @indexer.231 or @abi or @c2c"
+	mvn test -Dcucumber.filter.tags="@algod or @assets or @auction or @kmd or @send or @send.keyregtxn or @indexer or @rekey_v1 or @applications.verified or @applications or @applications.boxes or @compile or @dryrun or @indexer.applications or @indexer.231 or @abi or @c2c or @compile.sourcemap"
```

Comment:
@algochoi Sanity checking:  Does the PR deliberately remove the following `develop` addition?

https://github.com/algorand/java-algorand-sdk/blob/5db404e6fb310cf9c079693e0c09eb7a9863395c/Makefile#L6

Reply:
I haven't seen this line before in java-sdk, seems introduced in sdk testing improvement PR, what is this for?

my bad for missing this line.

Reply:
The integration tests seem to pass anyway - was this previously needed?

Reply:
I did search around, and found this: https://stackoverflow.com/questions/1873995/run-a-single-test-method-with-maven, and wondered if this line have any influence on integration test, for the test being pointed at is empty.

Reply:
@algochoi @ahangsu Let me know if the note helps clarify?

* `RunCucumberIntegrationTest` is the entry point for integration Cucumber test execution.
  * Admittedly, there's a bit of wiring between JUnit + Cucumber that I've not fully researched.  I suspect there's duplication `RunCucumberUnitTest`, but let's save that for another time. 
* Consequently, when executing `make integration`, the _only_ test Maven needs to target is `RunCucumberIntegrationTest`. 
* Without specifying the filter, `make integration` re-runs all non-Cucumber unit tests during integration test execution.  
  * So, the flag is not strictly necessary.  However, I think it's helpful to limit console output in order to disambiguate test output + speed up execution.
  * No Cucumber unit tests are run because no corresponding tags are provided in `-Dcucumber.filter.tags`.

Reply:
@michaeldiamant thanks for the explanation - I'll put the line back in 👍

Reply:
@algochoi Looks good as of https://github.com/algorand/java-algorand-sdk/pull/354/commits/9529ec1d4c154ac88dae026f1e68eac2b2afb22b.  Resolving.

Reply:
Tho its a bit late but, thanks for explaining!